index.html

```diff
@@ -15,47 +15,215 @@
+        </div>
+  
+        <div class="pg1content">
+          <p> I can help you build a product ,<br> feature or website Look through <br>some of my work and experience!<br> If you like what you see and have<br> a project you need coded,<br> 
+            don’t hesitate to contact me.</p>
+        </div>
+  
+        <div class="social">
+          <ul>
+           <li><a href="https://twitter.com/dvcorreia1984"> <img src="./Assets/social/twitter.svg" alt="Twitter"></a></li>
+           <li><a href="https://www.linkedin.com/in/dvcorreia/"><img src="./Assets/social/Linkedin icon.svg" alt="Linkedin"></a></li>
+           <li><a href="#"><img src="./Assets/social/medium.svg" alt="Medium"></a></li>
+           <li><a href="https://github.com/dvcorreia1984"><img src="./Assets/social/github.svg" alt="Github"></a></li>
+           <li><a href="#"><img src="./Assets/social/angellist.svg" alt="Angellist"></a></li>
+          </ul>
+        </div>
+
+        <div class="action">
+          <a href="#page2" class="action-button"><i class="fa fa-chevron-down" aria-hidden="true"></i></a>
+        </div>
+      </div>
+    </div>
+  </section>
 
-      <div class="page1bg"></div>
+  <section id="page2">
+    <div class="contentbox">
+      <header>
+        <h1 class="works">My Recent<br> Works</h1>
+        <div class="line"></div>
+      </header>
+      <div class=grid-container>
+          <div class="griditem">
+            <div class="placeholder"></div>
+              <div class=whitespace>
+                <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+                    <ul class="minimenu">
+                      <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+                      <li><div class="button2"><button2>CSS</button2><div><li>  
+                      <li><div class="button2"><button2>Javascript</button2><div><li>
+                      <li><div class="button2"><button2>HTML</button2><div><li>    
+                    </ul>
+                    <div class="tilebut"><button>See Project</button><div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
 
-      <div class="heading">
-        <h1>Hey there.<br>I am Victor.<br><span>I’m a software <br> developer</span></h1>
+        <div class="griditem">
+          <div class="placeholder"></div>
+            <div class=whitespace>
+              <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+                  <ul class="minimenu">
+                    <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+                    <li><div class="button2"><button2>CSS</button2><div><li>  
+                    <li><div class="button2"><button2>Javascript</button2><div><li>
+                    <li><div class="button2"><button2>HTML</button2><div><li>    
+                  </ul>
+                  <div class="tilebut"><button>See Project</button><div>
+              </div>
```

Comment:
- [ ] Kindly implement the button interactions (enable, hover, etc.) for all the buttons in the page and try to match the Figma design as well :+1: 

Here is the Figma design:

![hover_figma](https://github.com/dvcorreia1984/portfolio/assets/70483835/d6c38ed6-e22d-4c24-a47d-005baa8211a5)